### PR TITLE
feat(DC): DC supports new accepter resource to manage virtual interface

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:

- DC supports new accepter resource to manage virtual interface.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dc' TESTARGS='-run TestAccVirtualInterfaceAccepter_accepted'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dc -v -run TestAccVirtualInterfaceAccepter_accepted -timeout 360m -parallel 4
=== RUN   TestAccVirtualInterfaceAccepter_accepted
=== PAUSE TestAccVirtualInterfaceAccepter_accepted
=== CONT  TestAccVirtualInterfaceAccepter_accepted
--- PASS: TestAccVirtualInterfaceAccepter_accepted (20.92s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dc        20.967s
```

```
make testacc TEST='./huaweicloud/services/acceptance/dc' TESTARGS='-run TestAccVirtualInterfaceAccepter_rejected'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dc -v -run TestAccVirtualInterfaceAccepter_rejected -timeout 360m -parallel 4
=== RUN   TestAccVirtualInterfaceAccepter_rejected
=== PAUSE TestAccVirtualInterfaceAccepter_rejected
=== CONT  TestAccVirtualInterfaceAccepter_rejected
--- PASS: TestAccVirtualInterfaceAccepter_rejected (9.97s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dc        10.043s
```

### CODE REVIEW

File: docs/resources/dc_virtual_interface_accepter.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+subcategory: "Direct Connect (DC)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_dc_virtual_interface_accepter"
+description: "Manages a DC virtual interface accepter resource within HuaweiCloud."
+---
```
[the remark this answers is not in the document]
Comment (deer-hang) [reply docs/resources/dc_virtual_interface_accepter.md]: OK

File: huaweicloud/services/dc/resource_huaweicloud_dc_virtual_interface_accepter.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/dc/v3/interfaces"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// @API DC GET /v3/{project_id}/dcaas/virtual-interfaces/{interfaceId}
+// @API DC PUT /v3/{project_id}/dcaas/virtual-interfaces/{interfaceId}
```
Comment (Lance52259) [inline huaweicloud/services/dc/resource_huaweicloud_dc_virtual_interface_accepter.go:25]:
```suggestion
// @API DC GET /v3/{project_id}/dcaas/virtual-interfaces/{virtual_interface_id}
// @API DC PUT /v3/{project_id}/dcaas/virtual-interfaces/{virtual_interface_id}
```
https://support.huaweicloud.com/api-dc/UpdateVirtualInterface.html

Comment (deer-hang) [reply huaweicloud/services/dc/resource_huaweicloud_dc_virtual_interface_accepter.go:25]: Keep it consistent with other DC resources and do not modify it yet.

**Pull request merged**